salt/fileclient.py

```diff
@@ -551,7 +571,7 @@ def s3_opt(key, default=None):
                     https_enable=s3_opt("https_enable", True),
                 )
                 return dest
-            except Exception as exc:  # pylint: disable=broad-except
```

Comment:
This shouldn't be changing unless we are actually going to put a more applicable exception type here (which would be the preference).